src/Material/Button.elm

```diff
@@ -71,13 +79,89 @@ import Parts exposing (Indexed, Index)
 import Material.Helpers as Helpers
 import Material.Options as Options exposing (cs, when)
 import Material.Ripple as Ripple
+import Material.Options.Internal as Internal
 
 
 
--- MODEL
+{-| Opaque link property type
+-}
+type LinkProp = LinkProp
```

Comment:
Links occur also [in Layout](https://github.com/debois/elm-mdl/blob/v7/src/Material/Layout.elm#L481-L510). If we stick with `LinkProp` (see below), can we then refactor so that both use the same underlying implementation?